### PR TITLE
Checkout Page spec

### PR DESCRIPTION
**Pull request by XaosSintez**

### CODE REVIEW

File: spec/definitions/CheckoutPage.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  isActive:
+    description: If checkout page active
+    type: boolean
+  redirectUrl:
+    description: Checkout page url
+    type: string
+  redirectTimeout:
+    description: Checkout page redirect timeout
+    type: integer
+  customCustomerId:
```
Comment (slavcodev) [inline spec/definitions/CheckoutPage.yaml]: looks weird the property name for boolean type.

Comment (XaosSintez) [inline spec/definitions/CheckoutPage.yaml]: how about `allowCustomCustomerId` ?

File: spec/definitions/CheckoutPage.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    description: Checkout page plan ID
+    type: string
+    allOf:
+      - $ref: "#/definitions/ResourceId"
+  websiteId:
+    description: Checkout page website ID
+    type: string
+    allOf:
+      - $ref: "#/definitions/ResourceId"
+  uri:
```
Comment (slavcodev) [inline spec/definitions/CheckoutPage.yaml]: Actually there is not the `URI`, maybe another name? `uriPath`? `slug` (I'm not sure this name is clear)?

Comment (XaosSintez) [inline spec/definitions/CheckoutPage.yaml]: Changed to `uriPath` and added better description

**Pull request merged**